src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,13 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        StringBuilder reversedCut = new StringBuilder();
+        text = text.toLowerCase();
+        String str = text.replaceAll("\\W", "");
+        String firstCut = str.substring(0, str.length() / 2);
+        String secondCut = str.substring(str.length() / 2 + 1, str.length());
+        reversedCut.append(secondCut);
```

Comment:
you don't need to split it into two parts
just use reverse method on refactored code and compare them